test/core/utils/is-xhtml.js

```diff
@@ -0,0 +1,21 @@
+describe('axe.utils.isXHTML', function () {
```

Comment:
@rdeltour Have you tested this with Edge and IE? This seems like one of those things that might not work across the board :)

Reply:
Confirmed to work on Edge and IE too.
The logic relies on the fact that HTML parsers lower-case the tag names when parsing. It's been suggested as a possible reliable option by [Anne van Kesteren on Twitter](https://twitter.com/annevk/status/922805253585690624).